Applications/Utils/MeshEdit/removeMeshElements.cpp

```diff
@@ -101,12 +121,14 @@ int main (int argc, char* argv[])
 			INFO("%d %s elements found.", n_removed_elements, typeName.c_str());
 		}
 	}
-	if (matIDArg.isSet()) {
-		const std::vector<unsigned> vec_matID = matIDArg.getValue();
-		for (auto matID : vec_matID) {
-			const std::size_t n_removed_elements = ex.searchByMaterialID(matID);
-			INFO("%d elements with material ID %d found.", n_removed_elements, matID);
-		}
+
+	if (char_property_arg.isSet()) {
+		searchByProperty(property_name_arg.getValue(),
+		                 char_property_arg.getValue(), ex);
```

Comment:
Not this PRs fault, but `ex` is a poor name for an instance of `MeshLib::ElementSearch`.